linode/manager
# M3-6041: Add Awareness of DX tools (API-CLI ) - Create Linode flow

## Description 📝
Introduces API CLI Awareness Feature to cloud manager

**What does this PR do?**
Enables users copy  and execute dynamic cURL and CLI commands in create Linode flow.

## Preview 📷

https://user-images.githubusercontent.com/119517080/214948403-614f3fe7-55c2-4ddc-b5e9-1e91e7c69e52.mov

## How to test 🧪
- Click Create Linode 
- Click Create Using Command Line button
- Should be able to inline error messages
- Fill all the required fields to create Linode
- Now, click on Create Using Command Line button
- Should open API CLI Modal
- Validate the copy and Modal as per the design
- Copy the command (cURL / CLI )
- Open terminal and paste
- Update the command with token
- Execute and observe the Linode's page should appear created Linode

**How do I run relevant unit or e2e tests?**

- yarn test
- yarn cy:run
